apps/api/src/app/auth/google.strategy.ts

```diff
@@ -24,13 +25,7 @@ export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
     });
   }
 
-  public async validate(
-    request: any,
-    token: string,
-    refreshToken: string,
-    profile: Profile,
-    done: Function
-  ) {
+  public async validate(profile: Profile, done: Function) {
```

Comment:
I don't think this signature is still the same when these params are missing 🤔

Reply:
Also use `_` prefix?

Reply:
Actually this method is not used at all, at least my IDE marks this method as unused, but i didn't want to remove the code completely in this PR and instead focus on the typescript compile errors.

Is this method called in a very specific place which expects the method to have that exact signature it had before my refactoring?

Reply:
This method is indirectly called by extending the `PassportStrategy` (_Google Sign_ in in the cloud).